assets/css/viewer.css

```diff
@@ -43,10 +43,11 @@
 .viewer--sign-line {
   color: rgb(251, 220, 75);
   font-family: monospace;
-  font-size: 1.2em;
+  font-size: 1rem;
```

Comment:
Is this needed to override another font size somewhere between the body and this element? Without a specified font size it would already be `1rem`, by definition.